src/components/Checkout/BasketItem.vue

```diff
@@ -15,15 +15,20 @@
 					v-if="loan.loan.matchingText"
 					:matching-text="loan.loan.matchingText"
 				/>
+				<loan-reservation
+					:is-expiring-soon="loan.loan.loanFundraisingInfo.isExpiringSoon"
+					:is-funded="loan.isFunded"
+					:expiry-time="loan.expiryTime"
+				/>
 			</span>
 		</span>
 		<span class="small-3 show-for-small-only"></span>
 		<span class="small-9 medium-3 large-2 loan-res-price-wrapper">
-			<loan-reservation
+			<!-- <loan-reservation
 				:is-expiring-soon="loan.loan.loanFundraisingInfo.isExpiringSoon"
 				:is-funded="loan.isFunded"
 				:expiry-time="loan.expiryTime"
-			/>
+			/> -->
```

Comment:
Prob can remove this commented out code, unless you think it will come back here...